src/main/java/bookstore/service/user/UserService.java

```diff
@@ -0,0 +1,9 @@
+package bookstore.service.user;
+
+import bookstore.dto.user.UserRegistrationRequestDto;
+import bookstore.dto.user.UserResponseDto;
+import bookstore.exception.RegistrationException;
+
+public interface UserService {
+    UserResponseDto register(UserRegistrationRequestDto request) throws RegistrationException;
```

Comment:
```suggestion
    UserResponseDto register(UserRegistrationRequestDto request);
```

Reply:
Then I won't be able to throw an error in UserServiceImpl when registering with an existing email